packages/kit/src/core/prerender/prerender.js

```diff
@@ -109,7 +109,7 @@ export async function prerender({ config, entries, files, log }) {
 		if (seen.has(decoded)) return;
 		seen.add(decoded);
 
-		const file = decoded.slice(config.paths.base.length + 1);
+		const file = decoded;
```

Comment:
why did you remove this? no wonder the adapter-static tests are failing 😜

```suggestion
		const file = decoded.slice(config.paths.base.length + 1);
```

Reply:
Vite's `baseMiddleware` strips out the `base` path from the URL
https://github.com/vitejs/vite/blob/dfaeb2b7dc39822bf00fc97802b758b17129c37b/packages/vite/src/node/server/index.ts#L513
https://github.com/vitejs/vite/blob/dfaeb2b7dc39822bf00fc97802b758b17129c37b/packages/vite/src/node/server/middlewares/base.ts#L17